integ-test/src/test/java/org/opensearch/sql/calcite/standalone/calciteBuiltinFunctionsNullIT.java

```diff
@@ -0,0 +1,1003 @@
+package org.opensearch.sql.calcite.standalone;
```

Comment:
missing copyright, you can enable auto adding copyright in IDE.

Reply:
Done.